tests/flow/test_index_create.py

```
@@ -91,9 +91,12 @@ def test02_fulltext_index_creation_label_config(self):
         except ResponseError as e:
             self.env.assertIn("Can not override index configuration", str(e))
 
-        # drop L1:p6 - not indexed, so indices_deleted should be 0
-        result = self.graph.query("DROP FULLTEXT INDEX FOR (n:L1) ON (n.p6)")
-        self.env.assertEqual(result.indices_deleted, 0)
+        # drop L1:p6 - not indexed, so an error should be raised
+        try:
+            result = self.graph.query("DROP FULLTEXT INDEX FOR (n:L1) ON (n.p6)")
+            assert False
```

Ruff (0.15.10)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)